core/partitioning/shape_analysis.cpp

```diff
@@ -1,5 +1,6 @@
 #include "core/partitioning/shape_analysis.h"
 #include <ATen/ATen.h>
+#include <inttypes.h>

```

Comment:
Where does this file come from?

Reply:
I think this is a header from the standard C library. But actually it's not the one that was needed, so it's been replaced by `#include <cstdint>` in [this commit](https://github.com/pytorch/TensorRT/pull/1058/commits/fc13eb2c36f59e4ef122f28bd4476092370403cd).

This was added to address the type problem described in issue #856 :
```
input_shapes.push_back(util::toVec(util::toDims(c10::List<long int>({1}))));
 // replaced by 
input_shapes.push_back(util::toVec(util::toDims(c10::List<long long>({1}))));
```